src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const costs = 40;
+
+  if (days <= 0) {
```

Comment:
Instead of returning an error message, throw an error with 'throw new Error("Invalid days value");'.

Reply:
Not resolved.